src/design-system-team.md.njk

```diff
@@ -1,41 +1,48 @@
 ---
-title: GOV.UK Design System team
-description: about the GOV.UK Design System team and who works on it
+title: GOV.UK Design System and Prototype team
+description: about the GOV.UK Design System and Prototype team and who works on them
 layout: layout-single-page-prose.njk
 ---
 
-# GOV.UK Design System team
+# GOV.UK Design System and Prototype team
 
-The GOV.UK Design System team at the
-[Government Digital Service](https://www.gov.uk/government/organisations/government-digital-service) (GDS) maintains this design system.
+The GOV.UK Design System and Prototype team at the
+[Government Digital Service](https://www.gov.uk/government/organisations/government-digital-service) (GDS) maintains this design system. We also maintain the [GOV.UK Prototype Kit](https://govuk-prototype-kit.herokuapp.com/docs).
 
 If you want to contact the team you can
 [get in touch via email or Slack](/get-in-touch/).
 
-## Members
+## GOV.UK Design System team
+- Kelly Lee – Delivery Manager
+- Trang Erskine – Senior Product Manager
+
+### Design System squad
 
-- Amelia Phillips – User Researcher
-- Ben Surgison – Senior Node JS Developer
-- Brett Kyle – Frontend Developer
 - Calvin Lau – Senior Content Designer
 - Charlotte Downs – Interaction Designer
 - Chris Ballantine-Thomas – Senior Interaction Designer
 - Ciandelle Scollan – Interaction Designer
+- David Cox – Senior Accessibility Specialist
+- Imran Hussain – Community Designer
+- Katrina Birch – User Researcher
+- Kim 'beeps' Grey – Frontend Developer
+- Owen Jones – Senior Frontend Developer (Tech Lead for Design System squad)
+
+### Frontend squad
+
+- Brett Kyle – Frontend Developer
 - Claire Ashworth – Technical Writer
 - Colin Rotherham – Senior Frontend Developer
-- David Cox – Senior Accessibility Specialist
-- Imran Hussain – Community Manager
+- Oliver Byford – Lead Frontend Developer, Digital Service Platforms (Tech Lead for Frontend squad)
+- Romaric Pascal – Senior Frontend Developer
+
+## GOV.UK Prototype team
+
+- Amelia Phillips – User Researcher
+- Ben Surgison – Senior Node JS Developer
```

Comment:
Probably go with Node.js (or Node) to [match how we usually style it](https://gds-way.cloudapps.digital/manuals/programming-languages/nodejs/)

```suggestion
- Ben Surgison – Senior Node.js Developer
```